docs/nft-example.md

```diff
@@ -26,11 +26,13 @@ You have to pull the `CAP` submodule content as follows:
 
 ```sh
 yarn cap:init
+
+npm run cap:init
```

Comment:
That's alright, but the documentation already mentions that'll be using "yarn" as an example and the user can use npm or any other package manager if much prefer. So, to avoid having to show every single command for any other package manager, we accept that yarn is the base of the example, so no need to show the npm example.

"💡 During the guide we'll be using `yarn`, but this can be easily replaced by `npm`, if that's your preference"

```suggestion

```